Source/Fuse.Reactive.Bindings/Instance.API.uno

```diff
@@ -0,0 +1,356 @@
+using Uno;
+using Uno.Collections;
+using Uno.UX;
+
+using Fuse.Internal;
+
+namespace Fuse.Reactive
+{
+	public enum InstanceDefer
```

Comment:
Please add docs for the enum itself too. If it is only used one place, please point to the place it is used for docs.